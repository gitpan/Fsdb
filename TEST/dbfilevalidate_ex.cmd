prog='dbfilevalidate'
args=''
cmp='diff -c '
expected_exit_code=1
