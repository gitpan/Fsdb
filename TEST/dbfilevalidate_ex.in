#h sid cid
1 10
2
1 12
2 12
