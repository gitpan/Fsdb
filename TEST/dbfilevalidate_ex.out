#fsdb sid cid
1	10
2
# *** line above is missing field cid.
1	12
2	12
# *** dbfilevalidate: some lines had errors.
#   | dbfilevalidate
